### ISSUE TITLE
IPyWidgets don't work when debugging extension in VS Code Insiders

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#8617]
I found that IPyWidgets don't seem to work in VS Code insiders when debugging (while writing the IPyWidget tests).
For some reason the VSC Api isn't available to the code in the webview (acquireVSCodeApi) doesn't seem to return a valid value.

Creating this issue so we don't forget this, as its impossible to look into widget issues as a result.

Tried in debug mode on Mac, Windows & WSL with VS Code Insiders.
When not debugging, things work as expected.
Now, do the exact same thing in VS Code stable and things work (always, debugging, not debugging, windows, mac, wsl).
I think some change has been made in insiders, that's causing this to fall over.

**TLDR**
* We cannot debug ipywidgets in vscode insiders (it just doesn't work)

@rebornix @rchiodo @IanMatthewHuff /cc
Probably something for standup

**Comment by DonJayamanne** [comment]
Block this PR https://github.com/microsoft/vscode-jupyter/pull/8611

**Comment by greazer** [comment]
Rich said he was able to do this fine. Perhaps something to do with setting breakpoints?  Closing for now. Have a discussion with him.

**Comment by greazer** [comment]
@rchiodo believes this may be related to #8322.

**Comment by DonJayamanne** [comment]
Closing for now
